packages/requests/src/lib/requests-result.ts

```diff
@@ -15,6 +15,7 @@ import {
 
 export interface BaseRequestResult {
   staleTime?: number;
+  lastRequestTime?: number;
```

Comment:
What's the use case of this feature?

Reply:
![CleanShot 2022-12-11 at 15 09 24@2x](https://user-images.githubusercontent.com/13584853/206908673-6b8e7fd0-6a92-4e9b-9373-e142d58a8a02.png)

I use it to show when the data was last loaded. When the user clicks the refresh button the staleTime is reset and the data is loaded again. It's very useful to have it together with the other data.